test/ui-test/src/drivers/SquishDriver.py

```diff
@@ -243,6 +243,20 @@ def _find_link(objName: str, link: str):
     squish.uninstallSignalHandler(obj, "linkHovered(QString)", "_handle_link_hovered")
     return [-1, -1]
 
+
+def move_mouse_over_object(obj):
```

Comment:
Could we try using this method on `_find_link` since in some point `_find_link` also iterates through the object using `mouseMove`? Deos it make sense?

Reply:
I tried quickly, and it's a bit annoying because I don,t have a link, and I'd have to use `click_link` or make `find_link` not private.
Since the function really just needs to hover quickly, I think my version is simpler.